src/items/RqgItemSheet.ts

```diff
@@ -70,4 +73,46 @@ export class RqgItemSheet<
       });
     });
   }
+
+  protected async _onDrop(event: DragEvent): Promise<void> {
+    super._onDrop(event);
+    // Try to extract the data
+
+    if (
+      this.item.data.type === ItemTypeEnum.Cult ||
+      this.item.data.type === ItemTypeEnum.Rune ||
+      this.item.data.type === ItemTypeEnum.RuneMagic ||
+      this.item.data.type === ItemTypeEnum.Skill ||
+      this.item.data.type === ItemTypeEnum.SpiritMagic
+    ) {
```

Comment:
I'd rather not have ifs like that if possible. I guess the problem is that we want to extend for example `CultSheet` with both `RqgItemSheet` and some class with that `_onDrop` function (and you can only extend from one place). I see two possible ways - 
Either make a `RqgItemSheetDescribed` class that extends `RqgItemSheet` and adds the `_onDrop`. This is a bit clunky though in case we ever wanted to add something else.
Or explore mixins. I havn't used them, but I know Foundry itself uses [mixins for Documents](https://foundryvtt.com/api/ClientDocumentMixin.html). With a `hasDescription` mixin that implements the `_onDrop` function one could add that to the sheets that should have it.

This is typescripts documentation about mixins https://www.typescriptlang.org/docs/handbook/mixins.html

I did a quick search and found the starfinder system also uses mixins
https://github.com/wildj79/foundryvtt-starfinder/blob/development/src/module/actor/mixins/actor-conditions.js
https://github.com/wildj79/foundryvtt-starfinder/blob/6191ee7be6072d6e62fad8bd4207b6faba4cad38/src/module/utils/custom-mixer.js#L1
https://github.com/wildj79/foundryvtt-starfinder/blob/6191ee7be6072d6e62fad8bd4207b6faba4cad38/src/module/actor/actor.js#L20

Do you think this could be something to try out?